units/systemd-timesyncd.service.in

```diff
@@ -11,7 +11,6 @@ Documentation=man:systemd-timesyncd.service(8)
 ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
-RequiresMountsFor=/var/lib/systemd/clock
```

Comment:
hmm, why drop this? If /var is split out we still need that... Hmm, or is this implied by StateDirectory=? IIRC it currently isn't but probably should be...

Reply:
Indeed. Updated.